### PR TITLE
[Components with more than 1 service]  LPS-192496 Refactor ConfigurationModelIndexer component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-192496

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:usm

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:stable

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:usm

**Comment by Valmir-unicap** [comment]
ci:test:startup

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelIndexerHelper.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.configuration.admin.web.internal.util;
```
Comment (MrJohn1911) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelIndexerHelper.java:6]: Hey @Valmir-unicap please move it to the right package.

**Review by MrJohn1911 — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:stop

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:relevant

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:usm

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexerHelper.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.util.tracker.BundleTracker;
+import org.osgi.util.tracker.BundleTrackerCustomizer;
+
+/**
+ * @author Valmir Junior
+ */
+@Component(service = ConfigurationModelIndexerHelper.class)
+public class ConfigurationModelIndexerHelper {
+
```
Comment (rafaprax) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationModelIndexerHelper.java:37]: Hey @Valmir-unicap, in this case you don't need to create a helper class. You just need to inline the code extracted to the helper into the `ClusterConfigurationModelIndexer`. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
please resend

**Pull request closed without merging**